src/main/webapp/script.js

```diff
@@ -236,3 +255,39 @@ function getTooltipContent(node) {
 
   return content;
 }
+
+/**
+ * When a next/previous button is clicked, modifies the mutation index of the
+ * current graph to represent the new state. Then, the corresponding
+ * graph is requested from the server.
+ */
+function navigateGraph(amount) {
+  currGraphNum += amount;
+  if(currGraphNum < 0) {
```

Comment:
return out of it so you don't have to regenerate the graph? (i did this in mutBackend so if you don't it's nbd). But adding something to tell the user they're at the end would be nice!